File: src/main/java/Entities/Name.java
package Entities;

import org.json.JSONArray;
import org.json.JSONObject;

public class Name {

	private static final String GIVEN = "given";
	private static final String FAMILY = "family";
	private JSONArray name;

	public Name(String family, String given) {
		
		JSONArray ga = new JSONArray();
		ga.put(given);
		name = new JSONArray();
		JSONObject nameO = new JSONObject();
		nameO.put(FAMILY, family);
		nameO.put(GIVEN, ga);
		name.put(nameO);
	}
	
	public JSONArray getJSONObject() {
		return name;
	}
}

File: src/main/webapp/js/Integration/Name.js
function Name(family, given) {
	this.family = family;
	this.given = given;
};

Name.prototype.prettyPrint = function() {
	return "Given Name: " + this.given
	+ " Family Name: " + this.family;
};

Name.prototype.jsonPrint = function() {
	return { given: this.given,
				family: this.family
			};
};

module.exports = Name;
